srml/evm/src/lib.rs

```diff
@@ -27,13 +27,16 @@ use rstd::vec::Vec;
 use support::{dispatch::Result, decl_module, decl_storage, decl_event};
 use support::traits::{Currency, WithdrawReason, ExistenceRequirement};
 use system::ensure_signed;
+use sr_primitives::ModuleId;
 use sr_primitives::weights::SimpleDispatchInfo;
-use sr_primitives::traits::UniqueSaturatedInto;
+use sr_primitives::traits::{UniqueSaturatedInto, AccountIdConversion};
 use primitives::{U256, H256, H160};
 use evm::{ExitReason, ExitSucceed, ExitError};
 use evm::executor::StackExecutor;
 use evm::backend::ApplyBackend;
 
+const MODULE_ID: ModuleId = ModuleId(*b"py/ethvm");
```

Comment:
What does the “py/“ prefix mean in the module id?

Reply:
short for "parity"